common/autoware_ad_api_msgs/README.md

```diff
@@ -1,17 +1,25 @@
 # autoware_ad_api_msgs
 
-## InterfaceVersion
+## ResponseStatus
+
+This message is a response status commonly used in the service type API. For details, see [the response status][docs-response-status].
```

Comment:
Currently, it's linked not to your PR (https://github.com/autowarefoundation/autoware-documentation/pull/136) but to the old documentation.

For better maintainability, I think it's good to manage only high-level overviews in `autoware-documentation` and manage details alongside the implementations.

So, I mean, I recommend putting the details in this msg package and linking there from `autoware-documentation`.

@LalithVipulananthan What do you think about this?